library/oeUI/src/OemrUI.php

```diff
@@ -0,0 +1,290 @@
+            case "conceal":
+                $action_title = "Click to Hide"; // default needed for jQuery to function
+                $action_icon = "<i id='show_hide' class='fa fa-2x small fa-eye-slash' title='" . xla($action_title) . "'></i>";
+                break;
+            case "reveal":
+                $action_title = "Click to Show"; // default needed for jQuery to function
+                $action_icon = "<i id='show_hide' class='fa fa-2x small fa-eye' title='" . xla($action_title) . "'></i>";
+                break;
+            case "search":
+                $action_title = "Click to show search"; // default needed for jQuery to function
+                $action_icon = "<i id='show_hide' class='fa fa-search-plus fa-2x small' title='" . xla($action_title) . "'></i>";
+                break;
+            case "link":
+                if (strpos($action_href, 'http') !== false) {
+                    $target = '_blank';
+                } else {
+                    $target = '_self';
+                }
+                $action_title = ($action_title) ? $action_title : "Click to go to page";
+                $action_icon = "<a href='" . attr($action_href) . "' target = '" .attr($target)."' onclick='top.restoreSession()'><i id='advanced-action' class='fa fa-external-link fa-2x small' title='" . xla($action_title) ."' aria-hidden='true'></i></a>";
+                break;
+            case "back":
+                $action_title = ($action_title) ? $action_title : "Go Back";
+                if ($_SESSION ['language_direction'] == 'ltr') {
+                    $arrow_direction = 'fa-arrow-circle-left';
+                } elseif ($_SESSION ['language_direction'] == 'rtl') {
+                    $arrow_direction = 'fa-arrow-circle-right';
+                }
+                $action_icon = "<a href='" . attr($action_href) ."' onclick='top.restoreSession()'><i id='advanced-action' class='fa " . attr($arrow_direction) . " fa-2x small' title='" . xla($action_title) ."' aria-hidden='true'></i></a>";
+                break;
+            default:
+                $action_icon = '';
+        }
+        return $action_icon;
+    }
+    
+    /**
+    * Used to create the html string that will display the formatted help icon - fa-question-circle.
+    *
+    * @param int|bool  $display_help_icon
+    *
+    * @return string $help_icon that will output the help icon html string
+    *
+    */
+    
+    private function helpIcon($display_help_icon = '')
+    {
+        $display_help_icon = $this->display_help_icon;
+        if ($display_help_icon) {
+            if ($_SESSION ['language_direction'] == 'ltr') {
+                $help_icon_title = "To enable help - Go to the User Name on top right > Settings > Features > Enable Help Modal";
+            } elseif ($_SESSION ['language_direction'] == 'rtl') {
+                $help_icon_title = "To enable help - Go to the User Name on top left > Settings > Features > Enable Help Modal";
```

Comment:
wrap this with xl() function

Reply:
I thought we ought to translate and escape as close to the actual html output as possible

Reply:
see my comment I just posted